charts/atlantis/values.schema.json

```
@@ -0,0 +1,2360 @@
+      "additionalProperties":false,
+      "properties":{
+        "type":{
+          "description":"Service type.",
+          "type":"string",
+          "default":"NodePort",
+          "enum":[
+            "ClusterIP",
+            "NodePort",
+            "LoadBalancer"
+          ]
+        },
+        "annotations":{
+          "description":"Annotations to add to the service.",
+          "type":"object",
+          "additionalProperties":{
+            "type":"string"
+          }
+        },
+        "port":{
+          "description":"Port to expose on the service.",
+          "type":[
+            "string",
+            "integer"
+          ],
+          "default":80
+        },
+        "nodePort":{
+          "description":"Port to expose on the node when the service type is NodePort.",
+          "type":[
+            "string",
+            "integer",
+            "null"
+          ]
+        },
+        "targetPort":{
+          "description":"Port on the container to direct traffic to.",
+          "type":[
+            "string",
+            "integer"
+          ],
+          "default":4141
+        },
+        "loadBalancerIP":{
+          "description":"IP address to assign to load balancer (if supported).",
+          "type":[
+            "string",
+            "null"
+          ]
+        }
+      }
+    },
+    "podTemplate":{
+      "description":"Pod template configuration for pods.",
+      "type":"object",
+      "additionalProperties":false,
+      "properties":{
+        "annotations":{
+          "description":"Additional annotations to use for pods.",
+          "type":"object",
+          "additionalProperties":{
+            "type":"string"
+          }
+        },
+        "labels":{
+          "description":"Additional labels to use for pods.",
+          "type":"object",
+          "additionalProperties":{
+            "type":"string"
+          }
+        }
+      }
+    },
+    "statefulSet":{
+      "description":"StatefulSet configuration.",
+      "type":"object",
+      "additionalProperties":false,
+      "properties":{
+        "annotations":{
+          "description":"Additional annotations to use for StatefulSet.",
+          "type":"object",
+          "additionalProperties":{
+            "type":"string"
+          }
+        },
+        "labels":{
+          "description":"Additional labels to use for StatefulSet.",
+          "type":"object",
+          "default":{
+
+          },
+          "additionalProperties":{
+            "type":"string"
+          }
+        },
+        "securityContext":{
+          "type":"object",
+          "description":"Allow customizing securityContext.",
+          "$ref":"#/definitions/io.k8s.api.core.v1.PodSecurityContext"
+        },
+        "priorityClassName":{
+          "description":"Leverage a PriorityClass to ensure your pods survive resource shortages.",
+          "type":[
+            "string",
+            "null"
+          ]
+        },
```

I think this also needs to change?
```suggestion
        "priorityClassName":{
          "description":"Leverage a PriorityClass to ensure your pods survive resource shortages.",
          "type": "string"
        },
```

---

yes to be consistent i agree